ledfx/events.py

```diff
@@ -222,21 +225,37 @@ def __init__(
 
 
 class EffectSetEvent(Event):
-    """Event emitted when an effect is set or updated"""
+    """Event emitted when an effect is set"""
 
-    def __init__(self, effect_name, effect_id, effect_config, virtual_id):
+    def __init__(
+        self,
+        effect_id: str,
+        virtual_id: str,
+    ):
         super().__init__(Event.EFFECT_SET)
-        self.effect_name = effect_name
         self.effect_id = effect_id
-        self.effect_config = effect_config
+        self.virtual_id = virtual_id
+
+
+class EffectUpdatedEvent(Event):
```

Comment:
I was worried that this would be a change of interface behaviour with legacy implication, but as EffectSet was not called at all on update, despite the description, this is new event behaviour, rather than a mutation so no impact to existing usage.